src/app/view/components/login-as/login-as.spec.ts

```diff
@@ -115,15 +115,12 @@ describe('LoginAsComponent', () => {
     });
 
     it('should call redirect to the first porlet when login as happen', () => {
-        /*
-            TODO: need to revisit this component, too much going on to init
-        */
         comp.visible = true;
         comp.ngOnInit();
```

Comment:
You don't need to call

```
         comp.ngOnInit();
         fixture.detectChanges();
```

Because ```fixture.detectChanges();``` automatically run ```ngOnInit()```

Reply:
Clean this up in the other test too please.